pom.xml

```diff
@@ -530,6 +533,10 @@
                         <exclude>docs/demo/**/*.ipynb</exclude>
                         <exclude>docs/demo/**/*.zpln</exclude>
                         <exclude>**/src/main/resources/META-INF/services/*</exclude>
+                        <!-- Apache Rat excludes target folder for projects that are included by
+                        default. The reason we are explicitly excluding the target folder is because
+                        the tests-spark310+ project isn't included by default -->
+                        <exclude>**/target/**/*</exclude>
```

Comment:
Nit: I believe **/* is redundant, should be able to simplify this to `**/target/**`.